index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="description" content="You're invited to talk on Matrix">
     <meta name="viewport" content="width=device-width, user-scalable=no">
     <link rel="stylesheet" type="text/css" href="css/main.css">
+    <meta property="twitter:card" content="summary"/>
+    <meta property="twitter:image" content="https://matrix.org/blog/img/matrix-logo.png"/>
+	<meta property="twitter:site" content="@matrixdotorg"/>
+	<meta property="twitter:title" content="Matrix - An open standard for decentralised secure communication"/>
```

Comment:
```suggestion
	<meta property="twitter:title" content="Matrix - Decentralised and secure communication"/>
```

When being handed a matrix.to links, people will be expecting to use them for instant messaging. I don't think regular people using matrix.to links are interested in the standard itself.

I suggest this change for the sake of simplicity and concision.